app/models/claim_state_transition_reason.rb

```diff
@@ -2,60 +2,140 @@ class ClaimStateTransitionReason
   class ReasonNotFoundError < StandardError; end
   class StateNotFoundError < StandardError; end
 
-  attr_accessor :code, :description
+  attr_accessor :code, :description, :long_description
 
-  TRANSITION_REASONS = {
+  def self.klass_scope
+    [to_s.demodulize.underscore.downcase.to_sym]
+  end
+
+  def self.t(local_scope)
+    I18n.t(local_scope, scope: klass_scope)
+  end
+
+  TRANSITION_REASON_KEYS = {
```

Comment:
Given the new changes (with the localised values, couldn't this just be something like?

```ruby
TRANSITION_REASON_KEYS = YAML.load_file(<claim_transitions_locale_file>.yml).freeze
```

what's the reason for all the empty strings?

Reply:
they are loaded dynamically based on the key paths in the hash and the translation follows that format